02.calendar/calendar.rb

```diff
@@ -0,0 +1,74 @@
+# コマンドライン引数・オプションの処理
+require 'optparse'
+opt = OptionParser.new
+
+
+days = {}
+
+opt.on('-y [VAL]') {|v| days[:y] = v }
+opt.on('-m [VAL]') {|v| days[:m] = v }
+
+opt.parse!(ARGV)
+
+# 今日を定義する
+require 'date'
+
+today = Date.today
+
+
+# 年月を定義
+day = today.day
+
+if (days[:y].to_i) == 0
+  year = today.year
+end
+
+if (days[:m].to_i) == 0
+  month = today.month
+end
```

Comment:
`-m` と `-y` を指定するとエラーになるようになってしまったので調べてみたところ

ここの修正を間違えて指摘してしまっていました 🙇 

「削除しましょう」とお伝えしてしまったのですが、下記のように適切に改行を入れるように修正おねがいします 🙇 

```suggestion
if (days[:m].to_i) == 0
  month = today.month
else
  month = days[:m].to_i
end
```

year の代入も同様に `else` の処理を戻してください 🙏

Reply:
month, yearともに`else`の処理を戻しました。
ご確認ください！